packages/common/test/utils.test.ts

```diff
@@ -0,0 +1,34 @@
+import { getPackageJsonTasks } from '../src/utils';
+
+describe('Test common utils', () => {
+    test('Test package json task generation', () => {
+        let tasks = getPackageJsonTasks(false, true, '100', 'testApp-tile');
+        expect(tasks).toMatchInlineSnapshot(`
+            Object {
+              "start": "fiori run --open 'test/flpSandbox.html?sap-client=100#testApp-tile'",
+              "start-local": "fiori run --config ./ui5-local.yaml --open 'test/flpSandbox.html?sap-client=100#testApp-tile'",
+              "start-mock": "fiori run --config ui5-mock.yaml --open 'test/flpSandbox.html?sap-client=100#testApp-tile'",
+              "start-noflp": "fiori run --open 'index.html?sap-client=100'",
+            }
+        `);
+
+        tasks = getPackageJsonTasks(false, true);
+        expect(tasks).toMatchInlineSnapshot(`
+            Object {
+              "start": "fiori run --open 'test/flpSandbox.html'",
+              "start-local": "fiori run --config ./ui5-local.yaml --open 'test/flpSandbox.html'",
+              "start-mock": "fiori run --config ui5-mock.yaml --open 'test/flpSandbox.html'",
+              "start-noflp": "fiori run --open 'index.html'",
+            }
+        `);
+
+        tasks = getPackageJsonTasks(true, false, undefined, 'testApp-tile', undefined, 'testLocalStart.html');
+        expect(tasks).toMatchInlineSnapshot(`
+            Object {
+              "start": "echo \\\\\\"This application was generated with a local metadata file and does not reference a live server. Please add the required server configuration or start this application with mock data using the target: npm run start-mock\\\\\\"",
+              "start-local": "fiori run --config ./ui5-local.yaml --open 'testLocalStart.html#testApp-tile'",
+              "start-noflp": "echo \\\\\\"This application was generated with a local metadata file and does not reference a live server. Please add the required server configuration or start this application with mock data using the target: npm run start-mock\\\\\\"",
+            }
+        `);
+    });
```

Comment:
```suggestion
        describe('package.json task generation', () => {
        test('non empty sap-client is handled correctly', () => {
            let tasks = getPackageJsonTasks({
                localOnly: false,
                addMock: true,
                sapClient: '100',
                flpAppId: 'testApp-tile'
            });
            expect(tasks).toMatchInlineSnapshot(`
            Object {
              "start": "fiori run --open 'test/flpSandbox.html?sap-client=100#testApp-tile'",
              "start-local": "fiori run --config ./ui5-local.yaml --open 'test/flpSandbox.html?sap-client=100#testApp-tile'",
              "start-mock": "fiori run --config ./ui5-mock.yaml --open 'test/flpSandbox.html?sap-client=100#testApp-tile'",
              "start-noflp": "fiori run --open 'index.html?sap-client=100'",
            }
        `);
        });

        test('handles empty sap-client correctly', () => {
            let tasks = getPackageJsonTasks({ localOnly: false, addMock: true });
            expect(tasks).toMatchInlineSnapshot(`
            Object {
              "start": "fiori run --open 'test/flpSandbox.html'",
              "start-local": "fiori run --config ./ui5-local.yaml --open 'test/flpSandbox.html'",
              "start-mock": "fiori run --config ./ui5-mock.yaml --open 'test/flpSandbox.html'",
              "start-noflp": "fiori run --open 'index.html'",
            }
        `);
        });

        test('for local metadata file works correctly', () => {
            let tasks = getPackageJsonTasks({
                localOnly: true,
                addMock: false,
                sapClient: undefined,
                flpAppId: 'testApp-tile',
                localStartFile: 'testLocalStart.html'
            });
            expect(tasks).toMatchInlineSnapshot(`
            Object {
              "start": "echo \\\\\\"This application was generated with a local metadata file and does not reference a live server. Please add the required server configuration or start this application with mock data using the target: npm run start-mock\\\\\\"",
              "start-local": "fiori run --config ./ui5-local.yaml --open 'testLocalStart.html#testApp-tile'",
              "start-noflp": "echo \\\\\\"This application was generated with a local metadata file and does not reference a live server. Please add the required server configuration or start this application with mock data using the target: npm run start-mock\\\\\\"",
            }
        `);
        });
    });
```

Does this make more sense? 
The function `getPackageJsonTasks` accepts 6 parameters, 2 boolean and 4 optional strings. The contents of package.json is different based on the input. 
One test with 3 calls embedded with the description `'Test package json task generation'` could be changed to describe why we're choosing to test those 3 calls.

Reply:
It doesn't need to help me, it keeps the code base clean. I wouldn't call having simple, focused tests 'boiler-plate'.

>I think grouping them with comments makes more sense

Did I miss the comments?